new_arrivals_chi/app/config.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    DEBUG = False
```

Comment:
Since `DEBUG` is true for all but one category, you can default it to true here and just set it to false in `ProductionConfig`.

Reply:
Is `DEVELOPMENT` the environment flagging value?